h2o-core/src/main/java/hex/grid/Grid.java

```diff
@@ -475,4 +492,35 @@ public TwoDimTable createScoringHistoryTable() {
     ScoringInfo scoring_info = _scoring_infos != null && _scoring_infos.length > 0 ? _scoring_infos[0] : null;
     return ScoringInfo.createScoringHistoryTable(_scoring_infos, (scoring_info != null ? scoring_info.validation : false), (scoring_info != null ? scoring_info.cross_validation: false), m._output.getModelCategory(), (scoring_info != null ? scoring_info.is_autoencoder : false));
   }
+
+  /**
+   * Exports this Grid in a binary format using {@link AutoBuffer}. Related models are not saved.
+   *
+   * @param gridFilePath Full path to the name of the file this {@link Grid} should be saved to
+   * @throws IOException Error serializing the grid.
+   */
+  public void export_binary(final String gridFilePath) throws IOException {
```

Comment:
I think this should take just the parent dir as argument, since the path is always DIR + "/" + this.id

Reply:
Provides less control for naming, but not required now. Did as you suggested.